### PR TITLE
Add recipes for jupyterlab-renderers

### PR DESCRIPTION
**Pull request by jtpio**
Add recipes for the `jupyterlab-renderers` packages: https://github.com/jupyterlab/jupyter-renderers

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/jupyterlab-fasta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+about:
+  home: https://github.com/jupyterlab/jupyter-renderers
+  summary: Fasta renderer for JupyterLab
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - jasongrout
```
Comment (jtpio) [inline recipes/jupyterlab-fasta/meta.yaml:44]: cc @jasongrout let me know if it's ok to have you as maintainer here (and in the other recipes)

Comment (blink1073) [reply recipes/jupyterlab-fasta/meta.yaml:44]: Please add me as a maintainer as well

Comment (jtpio) [reply recipes/jupyterlab-fasta/meta.yaml:44]: Done, thanks!

**Review by blink1073 — APPROVED** [review]
Thanks!

**Comment by jtpio** [comment]
This should be ready to go.

**Comment by dhirschfeld** [comment]
Resolves https://github.com/jupyterlab/jupyter-renderers/issues/234

**Comment by dhirschfeld** [comment]
ping @conda-forge/staged-recipes

**Comment by chrisburr** [comment]
@jasongrout Can you comment on if you're happy to be listed as a maintainer on this?

(if there is already a comment/reaction somewhere for this, feel free to point me to it)

**Review by chrisburr — APPROVED** [review]

**Comment by jasongrout** [comment]
Yes, I'm happy to be a maintainer. Thanks.

**Pull request merged**